### DISCUSSION TITLE
Boolean control leads inversive in URL

### DISCUSSION THREAD

**Discussion by khats** [storybookjs/storybook#25034] [Help]
### Summary

Picked up random storybook from showcases where found story with boolean control:
https://5ccbc373887ca40020446347-veirmpnupy.chromatic.com/?path=/story/forms-input--template&args=hideLabel:!true

hideLabel is boolean and false by default in this story. 
After switching to true url changed to `hideLabel:!true`, but story hided label. 
After opening story in new frame url is https://5ccbc373887ca40020446347-veirmpnupy.chromatic.com/iframe.html?args=hideLabel:!true&id=forms-input--template&viewMode=story with `hideLabel:!true`, but label is visible.

I have same issue in my project in SB 7.6.0, assume during changing this control url should be updated to `hideLabel:!false` or `hideLabel:true`.  As i remember issues appeared somewhere in 7+ version

**Comment by JReinhold** [comment]
Thanks for raising this, I've created a bug report for this here: https://github.com/storybookjs/storybook/issues/25035
